packages/sdk/src/sdk/utils/location.ts

```diff
@@ -0,0 +1,34 @@
+import { LoggerService } from '../services'
+
+const ipApi = 'https://ipapi.co/8.8.8.8/json/'
```

Comment:
will this always resolve to Google MTV? On a Japan VPN it still resolves to Google MTV.

Reply:
oooomg good catch. was unintentional to leave the 8,8,8,8 in -- that will always put MTV. i was getting rate limited after testing. updated!